AWSSageMakerRuntimeTests/AWSSageMakerTests.swift

```diff
@@ -0,0 +1,105 @@
+//
```

Comment:
What is the difference between these tests and the ones above in `AWSSageMakerRuntimeTests.swift` ?

Reply:
One is for unit test and other for integration

Reply:
Can we move the unit tests to the "Unit Tests" directory then?

Reply:
Sorry my bad, both where same file. I removed one now.